core/services/feeds/orm.go

```diff
@@ -425,21 +425,16 @@ DO
 		status = (
 			CASE
 				WHEN job_proposals.status = 'deleted' THEN 'deleted'::job_proposal_status
+				WHEN job_proposals.status = 'approved' THEN 'approved'::job_proposal_status
 				ELSE EXCLUDED.status
 			END
 		),
-		external_job_id = (
```

Comment:
we shouldn't have to set the externalJobID of the proposals during an upsert. The upsert is used on `propose` from CLO. Proposal externalJobIDs should be updated depending on actual actions taken by nops, such as approval or cancellation.